Adds the possibility to define a custom registry for the ENS module (#3301)
ethereum/web3.js: docs/web3-eth-ens.rst

------------------------------------------------------------------------------

registryAddress
=====================

.. code-block:: javascript

    web3.eth.ens.registryAddress;

> The ``registryAddress`` property can be used to define a custom registry address in the case you are connected to an unknown chain.

Reviewer: ```suggestion
The ``registryAddress`` property can be used to define a custom registry address when you are connected to an unknown chain.
```

.. note::

>    If no address is defined will it try to detect the registry on the chain you are currently connected with and on the call of ``setProvider`` in the Eth module will it keep the defined address and use it for the ENS module.

Reviewer: ```suggestion
If no address is defined it will try to detect the registry on the chain you are currently connected with.  When  ``setProvider`` in the Eth module is called it will keep the defined address and use it for the ENS module.
```

Author: [As it looks like](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html) can we keep those spaces.
